### DISCUSSION TITLE
Gravitoelectromagnetism

### DISCUSSION THREAD

**Discussion by niklasrb** [einsteinpy/einsteinpy#638] [Ideas]
Hey everyone,
I got interested in GEM a while ago and used the symbolic module to do some calculations. I mostly followed [this](https://arxiv.org/pdf/2206.13946.pdf) review. Would you be interested in this kind of expansion? I could fix up the code and PR it. See [here](https://github.com/DMGW-Goethe/einsteinpy). Maybe someone finds this useful.

Best regards,
Niklas

**Comment by JeS24** [comment]
@niklasrb Hi. This looks interesting and it seems like you have added a lot of code related to GEM. As far as I am aware, GEM is mainly useful as a basic framework for studying weak field effects. Do you know, if this can be used for studying things like weak gravitational lensing in an academic, if not practical, context? And how does this fare against linearized gravity, which is admittedly a more general framework? I am mainly trying to gauge the range of applicability of GEM.

**Reply by JeS24** [reply]
Note that, even if GEM is useful for only academic purposes, I think, we can still add the expansion. Going through your changes, I see a lot of useful utility methods that can be used for other things as well. Some for instance can be useful vis-à-vis #405.

**Reply by niklasrb** [reply]
The context where I've seen this a lot is cosmology, for example [here](https://arxiv.org/pdf/gr-qc/9812046.pdf). Also maybe it is just becoming practical in GR simulations, see [here](https://arxiv.org/pdf/2211.08133.pdf). Other than that the applications are mostly theoretical GR [afaik](https://arxiv.org/pdf/1303.5612.pdf)

**Reply by niklasrb** [reply]
But okay, I will document and clean up the code a bit, create an example and then PR it

**Reply by JeS24** [reply]
@niklasrb I see. So, the context is mostly limited to theory, but in terms of the characterization of the Weyl Tensor and the entities that can be derived from it, this can come in handy. In particular, the "EBWeyl: ..." paper looks intriguing. I'd recommend continuing with the PR.
